decent-dao/fractal-interface
# Snapshot integration

## Description
1) This PR adds Snapshot integration for DAOs. When creating a DAO, users can configure a Snapshot ENS name for the DAO, and any proposals created within Snapshot will then be displayed in the DAO proposal list.


## Notes



## Issue / Notion doc (if applicable)
https://github.com/decent-dao/fractal-interface/issues/1154


## Testing
1) Go to [Snapshot ](https://demo.snapshot.org/#/) and set up a space with an ENS name. 
2) Within Fractal, create a DAO and set the ENS name used in step 1
3) Create some proposals in the Snapshot space and verify they show up in the DAO proposals list


## Screenshots (if applicable)
![image](https://github.com/decent-dao/fractal-interface/assets/84364476/06900bf4-db02-4239-8fd3-3d08e5d6e022)
